pkg/yurtctl/util/kubernetes/util.go

```diff
@@ -501,8 +500,8 @@ func RunServantJobs(cliSet *kubernetes.Clientset, tmplCtx map[string]string, nod
 	}
 	switch action {
 	case "convert":
-		servantJobTemplate = constants.ConvertServantJobTemplate
-		jobBaseName = ConvertJobNameBase
+		servantJobTemplate = nodeservant.ConvertServantJobTemplate
```

Comment:
can we try to use node-servant/job.go 
```
func RenderNodeServantJob(action string, tmplCtx map[string]string, nodeName string) (*batchv1.Job, error) 
```

I set that as the interface between modules, instead of reading the template directly.

Reply:
@adamzhoul  Oh, I didn't see this.  I agree this is a better way. 
I will adjust the `RunServantJobs` function as follow to suit the change.
```diff
// RunServantJobs launch servant jobs on specified nodes
- func RunServantJobs(cliSet *kubernetes.Clientset, tmplCtx map[string]string, nodeNames []string) error {
+ func RunServantJobs(cliSet *kubernetes.Clientset, job *batchv1.Job, nodeNames []string) error {
```

Reply:
@adamzhoul Hi, I just realize that the job name is related to the name of the node that it is running on. So the `RunServantJobs` should be able to get the job object for specific nodes. For example, the new `RunServantJobs` may look like:
```
// RunServantJobs launch servant jobs on specified nodes
func RunServantJobs(cliSet *kubernetes.Clientset, getJob func(nodeName string) *batchv1.Job, nodeNames []string) error {
	var wg sync.WaitGroup
	for _, nodeName := range nodeNames {
		job := getJob(nodeName)
		wg.Add(1)
		go func() {
			defer wg.Done()
			if err := RunJobAndCleanup(cliSet, job,
				WaitServantJobTimeout, CheckServantJobPeriod); err != nil {
				klog.Errorf("fail to run servant job(%s): %s",
					job.GetName(), err)
			} else {
				klog.Infof("servant job(%s) has succeeded", job.GetName())
			}
		}()
	}
	wg.Wait()
	return nil
}
```
As `RunServantJobs` is also used by `yurtctl revert`,  this change will lead to lots of changes. How about we leave a TODO comment here and refactor it in another PR?

Reply:
agree